static/src/js/components/DownloadHistory/__tests__/DownloadHistory.test.jsx

```diff
@@ -1,255 +1,226 @@
-      const { enzymeWrapper } = setup({
-        authToken: 'testToken',
-        earthdataEnvironment: 'prod',
-        retrievalHistory: [{
-          id: '8069076',
-          jsondata: {},
-          created_at: '2019-08-25T11:58:14.390Z',
-          collections: [{
-            title: 'Collection Title'
-          }, {
-            title: 'Collection Title Two'
-          }]
-        }],
-        retrievalHistoryLoading: false,
-        retrievalHistoryLoaded: true,
-        onDeleteRetrieval: jest.fn()
+    describe('when a retrieval exists with two collections', () => {
+      test('renders a table with the first collection title and count of others', () => {
+        setup({
+          earthdataEnvironment: 'prod',
+          retrievalHistory: [{
+            id: '8069076',
+            jsondata: {},
+            created_at: '2019-08-25T11:58:14.390Z',
+            collections: [{
+              title: 'Collection Title'
+            }, {
+              title: 'Collection Title Two'
+            }]
+          }],
+          retrievalHistoryLoading: false,
+          retrievalHistoryLoaded: true,
+          onDeleteRetrieval: jest.fn()
+        })
+
+        expect(screen.getByRole('table')).toBeInTheDocument()
+        expect(screen.getByRole('link', { name: 'Collection Title and 1 other collection' })).toHaveAttribute('href', '/downloads/8069076')
       })
-
-      const helmet = enzymeWrapper.find(Helmet)
-      expect(helmet.childAt(0).type()).toEqual('title')
-      expect(helmet.childAt(0).text()).toEqual('Download Status & History')
-      expect(helmet.childAt(1).props().name).toEqual('title')
-      expect(helmet.childAt(1).props().content).toEqual('Download Status & History')
-      expect(helmet.childAt(2).props().name).toEqual('robots')
-      expect(helmet.childAt(2).props().content).toEqual('noindex, nofollow')
-      expect(helmet.childAt(3).props().rel).toEqual('canonical')
-      expect(helmet.childAt(3).props().href).toEqual('https://search.earthdata.nasa.gov/downloads')
     })
 
-    test('renders links correctly when portals were used to place an order', () => {
-      const { enzymeWrapper } = setup({
-        authToken: 'testToken',
-        earthdataEnvironment: 'prod',
-        retrievalHistory: [{
-          id: '8069076',
-          jsondata: {
-            portal_id: 'test'
-          },
-          created_at: '2019-08-25T11:58:14.390Z',
-          collections: [{
-            title: 'Collection Title'
-          }]
-        }],
-        retrievalHistoryLoading: false,
-        retrievalHistoryLoaded: true,
-        onDeleteRetrieval: jest.fn()
-      })
-
-      expect(enzymeWrapper.find(Table).length).toBe(1)
-      expect(enzymeWrapper.find('tbody tr').length).toBe(1)
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('to')).toEqual({
-        pathname: '/downloads/8069076',
-        search: ''
+    describe('when portals were used to place an order', () => {
```

Comment:
This logic is part of PortalLinkContainer, not DownloadHistory, you don't need to test it here